theme/theme.config

```diff
@@ -62,7 +62,7 @@
 @sidebar    : 'pxt';
 @sticky     : 'pxt';
 @tab        : 'pxt';
-@transition : 'default';
+@transition : 'pxt';
```

Comment:
there are a handful more of these in this file that are set to default, and a LOT in the different targets; e.g. in arcade almost all modules just use 'default', including sidebar (which took a little bit to identify).

Not many have rules overridden in pxt, but it feels really confusing so unless there's opposition to it I'll go through the targets and make them all point at pxt for the themes so things don't randomly break

Reply:
yes we need to systematically fix this.